### PR TITLE
IMDb-Import

### PR DESCRIPTION
**Pull request by bartr**
Added IMDb-Import code

### CODE REVIEW

**Comment by aflinchb** [comment]
Should we remove one of the data folders so the json files aren't duplicated in 2 locations? (top level data folder and under src)

**Comment by bartr** [comment]
I like having data  at the root so people can easily find and explore the json, so I left the data duplicated but we could move Dockerfile to the root and update it to copy data and source separately. Thoughts?


Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by aflinchb** [comment]
Ah that's a good point.  I don't think it's necessary to move the Dockerfile, keeping as-is for ease of looking through the data makes sense to me. (Especially since the json files are small).

________________________________

File: imdb-import/src/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "imdb-import": {
+      "commandName": "Project",
+      "commandLineArgs": "bluebell kAYKP78ICFdi93UTlaJN4SiWsiBXw8JRcMkQWZAQlgM6z5pZChXzoYGIJzsCl0D2YKiqOYJQzLbmCq4HnMb7CQ== imdb movies2"
+    }
```
Comment (aflinchb) [inline imdb-import/src/Properties/launchSettings.json]: Does this need to be here? Or is there anyway to not have the key checked in?

**Comment by bartr** [comment]
That shouldn’t be there!

I think we should add that to .gitignore so it doesn’t get checked in and should validate all of our repos.

Good catch.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: imdb-import/src/Properties/launchSettings.json (hunk 1 above)
Comment (bartr) [reply imdb-import/src/Properties/launchSettings.json]: I removed the file and updated .gitignore so it won't get added back

**Pull request merged**